### PR TITLE
starting setup of ci to buld and upload to pypi and anaconda.org

### PR DESCRIPTION
**Pull request by christhechris**
Just a starting point, not full working yet.

using new nbdev release stuff: https://fastrelease.fast.ai

### CODE REVIEW

**Review by YiweiMao — APPROVED** [review]

**Comment by christhechris** [comment]
I think make file is functional, make release now does Conda and pypi, also add make changelog. Auto publish action not tested.

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+### New Features
+
+- More Updates with hardware, updated automated calibration, other tweaks. ([#13](https://github.com/openhsi/openhsi/pull/13)), thanks to [@christhechris](https://github.com/christhechris)
+  - - many updates to cal code for robustness.
+- tweaks to crop settings
+- add more flags tfm setup
+- add some plotting functions to plot raw data
+- update cal file with new property for cal luminance, previously hard coded.
+- fix nbs path issues from swap to notebooks in nbs folder.
+- fix radiance conversion for mode 4+. The reference files were not being binned etc, this is now now done during tfm_setup. This also removes redundant work in the dn2rad function. The current structure does not work for mode 5 this way, which needs to be re thought (due to wavelength def being smaller then full array after smile correction).
```
Comment (YiweiMao) [inline CHANGELOG.md:61]: I didn't realise we had a changelog. good idea. Looks like there are repeats though.

**Review by YiweiMao — APPROVED** [review]

**Comment by christhechris** [comment]
Yup, it was generated by fastrelease, didn’t check it for dups though, can
edit those out before merging.

**Comment by YiweiMao** [comment]
It probably draws from the commit messages? Do you have a PyPi account? I can add you to the project there so we can both test releases.

**Comment by christhechris** [comment]
Yup, I think it does. Also something about issues.

Pypi Username is christhechris

**Pull request closed without merging**